pkg/cmd/repo/create/create.go

```diff
@@ -257,15 +257,27 @@ func createRun(opts *CreateOptions) error {
 		repoToCreate = ghrepo.NewWithHost("", opts.Name, host)
 	}
 
+	input := repoCreateInput{
+		Name:              repoToCreate.RepoName(),
+		Visibility:        visibility,
+		OwnerLogin:        repoToCreate.RepoOwner(),
+		TeamSlug:          opts.Team,
+		Description:       opts.Description,
+		HomepageURL:       opts.Homepage,
+		HasIssuesEnabled:  opts.EnableIssues,
+		HasWikiEnabled:    opts.EnableWiki,
+		GitIgnoreTemplate: gitIgnoreTemplate,
+		LicenseTemplate:   repoLicenseTemplate,
+	}
+
+	httpClient, err := opts.HttpClient()
+	if err != nil {
+		return err
+	}
+
 	var templateRepoMainBranch string
-	// Find template repo ID
 	if opts.Template != "" {
-		httpClient, err := opts.HttpClient()
-		if err != nil {
-			return err
-		}
-
-		var toClone ghrepo.Interface
+		var templateRepo ghrepo.Interface
 		apiClient := api.NewClientFromHTTP(httpClient)
 
 		cloneURL := opts.Template
```

Comment:
I know that this was already in place but I think it would be clearer to rename this to `templateRepoURL` or something similar since `toClone` was renamed to `remplateRepo`.